### ISSUE TITLE
InsertLookupBoMapping.sql & MAXIMS_RefDataMSQ05.sql

### ISSUE THREAD

**Issue by kakoni** [IMS-MAXIMS/openMAXIMS#24]
Its not clear what are these SQL scripts.
`Creating the openMAXIMS database.pdf` doesnt mention these files.

Does MAXIMS_RefDataMSQ05.sql include newer reference/demo data and should be used instead of  Reference-Data-scripts-rel147-openMAXIMS-utf.sql?

**Comment by ThiyagarajanJayaraman** [comment]
Yes, Please use MAXIMS_RefDataMSQ05.sql for SQL Server and MAXIMS_RefDataORA.sql for Oracle.

**Comment by kakoni** [comment]
Ok. And whats the role for InsertLookupBoMapping.sql?
